src/test/java/stest/tron/wallet/onlineStress/TestStorageAndCpu.java

```diff
@@ -45,109 +64,118 @@ public void beforeSuite() {
+    byte[] contractAddress = PublicMethed.deployContract(contractName,abi,code,
+        "",maxFeeLimit,
+        0L, 100,null,testKey002,fromAddress,blockingStubFull);
+    try {
+      Thread.sleep(30000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
+    SmartContract smartContract = PublicMethed.getContract(contractAddress,blockingStubFull);
+    String txid;
+
+    Integer i = 1;
+    while (i++ < 40) {
+      String initParmes = "\"" + "930" + "\"";
+      /*      txid = PublicMethed.triggerContract(contractAddress,
+          "testUseCpu(uint256)", "9100", false,
+          0, maxFeeLimit, fromAddress, testKey002, blockingStubFull);*/
+      txid = PublicMethed.triggerContract(contractAddress,
+          "storage8Char()", "", false,
+          0, maxFeeLimit, fromAddress, testKey002, blockingStubFull);
+      //storage 9 EnergyUsageTotal is  211533, 10 is 236674, 5 is 110969,21 is 500000
+      /*      txid = PublicMethed.triggerContract(contractAddress,
+          "testUseStorage(uint256)", "21", false,
+          0, maxFeeLimit, fromAddress, testKey002, blockingStubFull);
+      logger.info("i is " +Integer.toString(i) + " " + txid);*/
+      txidList.add(txid);
       try {
-        Thread.sleep(3000);
+        Thread.sleep(50);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2925))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1678&issues=AWbiJDiULpqqSUlcrGtF&open=AWbiJDiULpqqSUlcrGtF)

[ISSUE_KEY]: AWbiJDiULpqqSUlcrGtF